frontstage_api/controllers/case_controller.py

```diff
@@ -131,15 +131,19 @@ def build_full_case_data(case):
     return survey_data
 
 
-def calculate_case_status(case):
+def calculate_case_status(case, collection_instrument_type):
     logger.debug('Getting the status of case')
+    status = 'Not started'
     case_group_status = case.get('caseGroup', {}).get('caseGroupStatus')
 
-    status = {
-        'COMPLETE': 'Complete',
-        'COMPLETEDBYPHONE': 'Completed by phone',
-        'INPROGRESS': 'Downloaded'
-    }.get(case_group_status, 'Not Started')
+    if case_group_status == 'COMPLETE':
```

Comment:
the decisions of what text to render shouldn't really live in the api, the view should be deciding what to text render based on the data it's given

Reply:
in the interest of getting stuff in i've raised and issue for this https://github.com/ONSdigital/ras-frontstage-api/issues/45